File: java/logica_y_validaciones/validar_longitud_del_nombre.java
package logica_y_validaciones;
public class validar_longitud_del_nombre {
    public static void main(String[] args) {
        String nombre = "Ana";
        if (nombre.length() >= 3) {
            System.out.println("Nombre válido.");
        } else {
            System.out.println("El nombre es demasiado corto.");
        }
    }
}

File: visual_basic/condicionales/validar_longitud_del_nombre/validar_longitud_del_nombre.vb
Imports System
Module validar_longitud_del_nombre
    Sub Main(args As String())
        Dim nombre As String
        Console.Write("Ingrese el nombre: ")
        nombre = Console.ReadLine()
        If nombre.Trim().Length >= 3 Then
            Console.WriteLine("Nombre válido ✅")
        Else
            Console.WriteLine("Error: El nombre debe tener al menos 3 caracteres ❌")
        End If
    End Sub
End Module